### DISCUSSION TITLE
How to open other data instead of database 0

### DISCUSSION THREAD

**Discussion by sunjianan9900** [ra1u/redis-dart#66] [Q&A]
How to open other data instead of database 0 ?

**Comment by ra1u** [comment]
Hi @sunjianan9900  If you are asking how to send command [SELECT](https://redis.io/commands/select/) you can try with 

~~~dart
var r = await cmd.send_object(["SELECT", db_index])
~~~

In such case you will need as many tcp connection as you are using database indexes.

**Reply by sunjianan9900** [reply]
Thanks for the reply, now I introduced the expiration timestamp, so is it right to write it that way?

 ```
command.send_object([
        "SET", "data", "EX", 60 * 60 * 24 * 30, db_index
      ]).then((var response) {
        assert(response == 'OK');
      });
```

**Reply by ra1u** [reply]
I am not sure that redis SET enables selecting db_index. Can you check manual of command set from redis?
